notebooks/book2/02/sub_super_gauss_plot.ipynb

```diff
@@ -0,0 +1,443 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    from scipy.stats import uniform, laplace, norm
</pre><p>In the three separate plots you use <code>jax.random.normal</code> etc. Why do we need <code>scipy.stats</code>?You should just use <code>jax.random.*</code> for all the plots in this notebook.     </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
okay